src/css/vars.scss

```diff
@@ -1,21 +1,28 @@
 // Colors
 
 $aqua-marine: #48dbb4;
-$white: #f2f2f2;
-$black: #0A0806;
+$mint-10: #EDFBF7;
+$mint-20: #C8F4E9;
+$mint-30: #91E9D2;
+$mint-40: #48DBB4;
+$mint-50: #12C0A1;
+$mint-60: #1BA58C;
 $violet: #7850CD;
 $brass: #AC6E16;
+$black: #0A0806;
+$white: #FFFFFF;
+$grey-light: #f2f2f2;
 $grey-10: #F4F4F4;
 $grey-20: #E6E6E6;
 $grey-30: #c4c4c4;
 $grey-40: #AAAAAA;
-$grey-50: #818181;
+$grey-50: #818181; 
```

Comment:
Please remove the `space` at the end of the line that can affect the lint issue for the deploy.